packages/html-webpack-plugin/README.md

```diff
@@ -0,0 +1,42 @@
+# @talend/html-webpack-plugin
+
+This plugin is to provide some default [hooks](https://github.com/jantimon/html-webpack-plugin#events) into `html-webpack-plugin`
+
+## options
+
+The plugin support the following options:
+
+| name | description |
+| -- | -- |
+| loadCSSAsync | Put all CSS in the body and use a trick to not block the rendering of the App |
+| bodyBefore | Usefull to add some assets (script / link) into the body before the app |
+| headerStyles | Usefull to add styles in the HEAD (to display a loader) |
+
+example:
+
+```javascript
+const TalendHTML = require('@talend/html-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const Loader = require('@talend/react-components/lib/AppLoader/constant').default;
+const ICON = "url('data:image/svg+xml;base64,PHN2...')";
+const EXTENSION = process.env.NODE_ENV === ENV.DEV ? 'js' : 'min.js';
+
+//... plugins
+    new HtmlWebpackPlugin({
+        // ... your options
```

Comment:
Maybe we can also add the loader key here, along with a very short index.html to show how you can insert the loader markup? Just to make sure we all use it the same way